backend/src/custom-server.js

```diff
@@ -44,48 +45,20 @@ app.get('/public', async (req, res) => {
   try {
     const { numMessages = 50 } = req.body;
 
-    const messages = await db.message.findAll({
-      where: { flagged: false },
-      order: [['createdAt', 'DESC']],
-      limit: numMessages,
-      include: [db.user],
-    });
+    const messages = await messageRepository.getAll(numMessages, true);
 
-    return res.send(messages.map((msg) => ({
-      text: msg.text,
-      pubDate: msg.createdAt,
-      userId: msg.user.id,
-      username: msg.user.username,
-      email: msg.user.email,
-    })));
+    return res.send(messages);
   } catch (err) {
     return res.status(500).send(err.message);
   }
 });
 
 async function getTimeline(userId, numMessages) {
-  const user = await db.user.findByPk(userId);
-
-  const messages = await db.message.findAll({
-    where: {
-      flagged: false,
-      [db.Sequelize.Op.or]: [
-        { userId: await user.getFollow().map((flw) => flw.id) },
-        { userId: user.id },
-      ],
-    },
-    limit: numMessages,
-    include: [db.user],
-    order: [['createdAt', 'DESC']],
-  });
-
-  return (messages.map((msg) => ({
-    text: msg.text,
-    pubDate: msg.createdAt,
-    userId: msg.user.id,
-    username: msg.user.username,
-    email: msg.user.email,
-  })));
+  const user = await userRepository.getById(userId);
+
+  const messages = await messageRepository.getTimeline(user, numMessages);
+
+  return messages;
```

Comment:
Flueknep, hvis du magter at ændre det... return await med det samme i stedet for at lav en lokalvariable